cfgov/jinja2/v1/_includes/organisms/filterable-list-controls.html

```diff
@@ -254,6 +254,8 @@
                             'sub_heading': sub_heading,
                             'body': parse_links(post.preview_description),
                             'links': links or null,
+                            'should_link_image': controls.should_link_image
+                                                if controls.should_link_image else false
```

Comment:
Fixed in https://github.com/cfpb/cfgov-refresh/pull/2753/commits/49fae6d0b0012d1922277b8c1d24efc9045f2723.